### PR TITLE
Always deploy with governor as admin and transfer ownership after

### PR DESCRIPTION
**Pull request by nepoche**

### CODE REVIEW

**Review by drewstone — APPROVED** [review]
LGTM

File: packages/bridges/src/SignatureBridgeSide.ts [hunk 1]
```diff
@@ -18,60 +18,52 @@ export class SignatureBridgeSide implements IBridgeSide {
...
+
+  public static async connectGovernor(
+    contractAddress: string,
+    provider: ethers.providers.Provider,
+    systemSigningFn: SystemSigningFn
+  ) {
+    const deployedBridge = SignatureBridge__factory.connect(contractAddress, provider);
+    const bridgeSide = new SignatureBridgeSide(deployedBridge, systemSigningFn);
     return bridgeSide;
   }
```
Comment (shekohex) [inline packages/bridges/src/SignatureBridgeSide.ts:68]: A few comments here would make things easier to understand. Like what are these two method does, and what is the difference? a use case(s) in the comments would make much sense to understand the difference between both.

File: test/vbridge/signatureVBridge.test.ts [hunk 2]
```diff
@@ -139,17 +141,51 @@ describe('2-sided multichain tests for signature vbridge', () => {
...
+      };
+
+      const deploymentConfig: DeployerConfig = {
+        [chainID1]: hardhatWallet1,
+        [chainID2]: ganacheWallet2,
+      };
+
+      const initialGovernorsConfig: GovernorConfig = {
+        [chainID1]: '0x2B5AD5c4795c026514f8317c7a215E218DcCD6cF',
+        [chainID2]: '0x2B5AD5c4795c026514f8317c7a215E218DcCD6cF',
```
Comment (shekohex) [inline test/vbridge/signatureVBridge.test.ts:168]:
```suggestion
        [chainID1]: governorAddress,
        [chainID2]: governorAddress,
```

**Review by shekohex — CHANGES_REQUESTED** [review]
Looks great!
I left a few comments to improve this a bit, like adding docs and improve some parts in the tests.

File: packages/bridges/src/SignatureBridgeSide.ts (hunk 1 above)
Comment (nepoche) [reply packages/bridges/src/SignatureBridgeSide.ts]: Added some comments explaining the use of both connectMocked and connectGovernor.

**Review by shekohex — APPROVED** [review]
LGTM!

**Pull request merged**